public/js/wyr-single.js

```diff
@@ -190,3 +135,97 @@ async function createComment() {
     console.log(err);
   }
 }
+
+/* ADDING AND DELETING QUESTION UI */
+const addQuestionBtn = document.getElementById("add-question-button");
+const questionModal = document.getElementById("add-question-modal");
+const questionForm = document.getElementById("addQuestionForm");
+const addQuestionCloseModal = document.getElementById(
+  "addQuestion-close-modal"
+);
+const deleteQuestionBtn = document.getElementById("deleteQuestionBtn");
```

Comment:
these are the elements that are selected for the below functions of adding question and deleting questions. I'm sure it's not great practice to have these things below vs on the top of the page, but I put it here so these elements are closer to the functions